sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/KeyVaultCertificateWithPolicy.java

```diff
@@ -28,14 +28,6 @@
         super();
     }
 
-    /**
-     * Get the certificate properties.
-     * @return the certificate properties.
-     */
-    public CertificateProperties getProperties() {
```

Comment:
Is this still on `KeyVaultCertificate`? It should be.

Reply:
yes it is.